.mailmap

```diff
@@ -276,6 +276,7 @@ MingXin Liu <mingxin.liu@kylin-cloud.com> <mingxinliu@ubuntukylin.com>
 Mingqiao Wu <wumingqiao@inspur.com>
 Mingyue Zhao <zhao.mingyue@h3c.com>
 Mykola Golub <mgolub@mirantis.com> <mgolub@zhuzha.mirantis.lviv.net>
+Mykola Golub <to.my.trociny@gmail.com>
```

Comment:
Nit: If `to.my.trociny@gmail.com` is your primary email id, why can't you map your other email ids (mgolub@mirantis.com and mgolub@zhuzha.mirantis.lviv.net) to the primary email id. For example, see Sage Weil's or Kefu Chai's entry.

Reply:
My idea was that all commits I had done working at Mirantis would remain associated with mgolub@mirantis.com, and only new commits would be associated with to.my.trociny@gmail.com, which is unaffiliated. I am not sure it is possible though.

Reply:
@trociny just add a .peoplemap entry as well, that will ensure that old commits are still associated

Reply:
@theanalyst Thanks, updated.